providers/jobids.rb

```diff
@@ -1,3 +1,4 @@
 action :reset do
-  node.normal['gecos_ws_mgmt'][new_resource.recipe][new_resource.resource]['job_ids'] = []
-end
\ No newline at end of file
+  node.normal['gecos_ws_mgmt'][new_resource.recipe][new_resource.resource][
+    'job_ids'] = []
```

Comment:
Strange line splitting

Reply:
Rubocop doesn't detect it as an offense to the Ruby style guide. I tried other ways of splitting this type of lines and Rubocop detected them as an offense.

Reply:
ok